examples/routing_service/routing_service_msql_adapter/cpp/rs_main/CMakeLists.txt

```diff
@@ -0,0 +1,91 @@
+#
```

Comment:
Please, apply the same changes suggested for the other `CMakeLists.txt`

Reply:
Done. Tested that both rs_dll adn rs_main build. Will commit all changes at once.